Services/Init/classes/class.ilStartUpGUI.php

```diff
@@ -528,7 +528,7 @@ protected function initStandardLoginForm(): ilPropertyFormGUI
         $pi->setRetype(false);
         $pi->setSkipSyntaxCheck(true);
         $pi->setSize(20);
-        $pi->setDisableHtmlAutoComplete(false);
+        $pi->setDisableHtmlAutoComplete(true);
```

Comment:
I stumbled about this while working on #6031. This was disabled here based on a JF decision some years ago. Maybe we'll have to discuss this again if we like to change the behaviour.

Reply:
> I stumbled upon this while working on #6031. This was disabled here based on a JF decision some years ago. Maybe we'll have to discuss this again if we like to change the behaviour.

To document the findings of @mjansenDatabay: https://docu.ilias.de/goto_docu_wiki_wpage_3106_1357.html